applications: nrf_desktop: Document SUIT support (#15851)
nrfconnect/sdk-nrf: scripts/hid_configurator/README.rst

   For more detailed information about LED stream functionality, see the `Playing LED stream`_ section.

#. If you want to perform :ref:`ug_nrf54h20_suit_dfu`, you must also install the SUIT generator Python package.

>    The package source files are available in the :file:`ncs/modules/lib/suit-generator` directory of the nRF Connect SDK.

Reviewer: ```suggestion
   The package source files are available in the :file:`ncs/modules/lib/suit-generator` directory of the |NCS|.
```

   For more detailed information about LED stream functionality, see the `Playing LED stream`_ section.

#. If you want to perform :ref:`ug_nrf54h20_suit_dfu`, you must also install the SUIT generator Python package.

>    The package source files are available in the :file:`ncs/modules/lib/suit-generator` directory of the nRF Connect SDK.

Reviewer: ```suggestion
   The package source files are available in the :file:`ncs/modules/lib/suit-generator` directory of the |NCS|.
```

If the process is to be continued, the script uploads the image data to the device.
When the upload is completed, the script reboots the device.

Customize the command with path to the DFU update file (``UPDATE_IMAGE_PATH``).

> The used file type depends on the following SoC series:

Reviewer: ```suggestion
The used file type depends on the following devices:
```

* nRF52 and nRF53 Series use the :file:`dfu_application.zip` file.

> * nRF54 Series use the :file:`*.suit` file.

Reviewer 2: `nRF54H Series` (nRF54L uses MCUboot actually)
